transactions/templates/fundTransfer.html

```diff
@@ -1,16 +1,20 @@
 {% extends "base.html" %}
 {% block content %}
-<div class="form-group">
-  <form action='./fundTransfer' method="post">
-              {% csrf_token %}
-      <label>From Account:</label>
-      <input type="number" id="fromAccount" class="form-control" name="fromAccount"><br><br>
-      <label>To Account:</label>
-      <input type="number" id="toAccount" class="form-control" name="toAccount"><br><br>
-      <label>Amount:</label>
-      <input type="number" id="transferAmount" step="0.01" class="form-control" name="transferAmount"><br><br>
-      <input type="submit" value="Submit" class="btn btn-primary float-right app-submit">
-  </form>
+<div>
+    <div>
+        <h2>Fund Transfer</h2><br><br>
+    </div>
+    <div class="form-group">
+      <form action='./fundTransfer' method="post">
+                  {% csrf_token %}
+          <label>From Account:</label>
```

Comment:
This form needs checks, right? Why can't we use Django forms here?
Also, can we change this so that customer selects from available accounts. I think that is expected...

Reply:
It is using django form, just the UI looks different in template. In views you can see forms being used. This is just a different way of showing form